README.md

```diff
@@ -1,13 +1,42 @@
-Subscription promotions tool
-====================
+# Subscription promotions tool
```

Comment:
for a nicer view just look at https://github.com/guardian/memsub-promotions/tree/jd-readme/README.md